pkgs/by-name/pi/pista/package.nix

```
@@ -0,0 +1,34 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  darwin,
+  zlib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pista";
+  version = "unstable-2023-02-20";
```

```suggestion
  version = "0.1.5-unstable-2023-02-20";
```
New policy, unstable versions should report the last stable version they are based on.